Simulator/Utils/RNG/Norm.h

```diff
@@ -55,41 +55,51 @@
 */
 class Norm : public MTRand {
 public:
-    inline virtual ~Norm() {}
-
-  /*!
-    The constructor allows specification of the mean,
-    variance (default zero and one, respectively), and initial seed
-    for the random number generator. Once created, a Norm object
-    cannot have its mean or variance changed.
-    @param m mean
-    @param s variance
-    @param seed seed for random number generator
-  */
-  Norm(BGFLOAT m = 0.0, BGFLOAT s = 1.0, uint32_t seed = 0)
-    : MTRand(seed), odd(true), mu(m), sigma(s) {}
-
-  /*!
-    This method makes instances functors; it returns normally
-    distributed random numbers. Just a cute way of doing things.
-    @return pseudorandom number drawn from a normal distribution.
-  */
-  virtual BGFLOAT operator() (void);
+   virtual ~Norm();
+
+   ///  Creates an instance of the class.
+   ///
+   ///  @return Reference to the instance of the class.
+   static MTRand *Create() { return new Norm(); }
+
+   /*!
+      The constructor allows specification of the mean,
+      variance (default zero and one, respectively), and initial seed
+      for the random number generator. Once created, a Norm object
+      cannot have its mean or variance changed.
+      @param m mean
+      @param s variance
+      @param seed seed for random number generator
+   */
+   Norm(BGFLOAT m = 0.0, BGFLOAT s = 1.0, uint32_t seed = 0);
+
+   /*!
+      This method makes instances functors; it returns normally
+      distributed random numbers. Just a cute way of doing things.
+      @return pseudorandom number drawn from a normal distribution.
+   */
+   virtual BGFLOAT operator() (void) override;
+
+   /// Allow Norm re-seeding (with same behavior as initializers)
+   virtual void seed(BGFLOAT m, BGFLOAT s, uint32_t seed);
+   virtual void seed(uint32_t seed) override;
+   virtual void seed() override;
+
 private:
-  // Additional state information
+   // Additional state information
 
-  /*! Which of the pair of pseudorandom numbers was last
-    returned. Says whether we should calculate this time  */
-  bool odd;
```

Comment:
Change member variable names to be consistent with our standards.

Reply:
Added `_` after all member variables, but didn't change the names further,